setup.py

```diff
@@ -364,6 +364,11 @@ def setup_package():
         url = "https://www.numpy.org",
         author = "Travis E. Oliphant et al.",
         download_url = "https://pypi.python.org/pypi/numpy",
+        project_urls={
+            "Bug Tracker": "https://github.com/numpy/numpy/issues",
+            "Documentation": "https://docs.scipy.org/doc/numpy/",
```

Comment:
@rgommers Is this the best place to view the docs?

Reply:
yes that is the best one, that's the full manual of the latest release always.